Assets/MixedRealityToolkit-Examples/Demos/UX/TwoHandManipulation/ManipulationExample.unity

```diff
@@ -0,0 +1,2182 @@
+%YAML 1.1
```

Comment:
When running in VR, the demo scene objects appear below the floor plane. 0,0,0 is on the floor in VR vs at the head in HoloLens

Reply:
I think we will have this issue until we port SceneContentAdjuster script to vNext. Since the primary use of this interaction is in HoloLens, I would keep it at 0,0,0 for now.